/**
 * Copyright (c) 2015 - The CM Authors <legal@clickmatcher.com>
 *   All Rights Reserved.
 *
 * This file is CONFIDENTIAL -- Distribution or duplication of this material or
 * the information contained herein is strictly forbidden unless prior written
 * permission is obtained.
 */
package cm;

message ReportParams {
  required string customer = 1;
  required string report = 2;
  optional uint64 from_unixmicros = 3;
  optional uint64 until_unixmicros = 4;
  optional string params = 5;
}
